boot_default/config.txt

```diff
@@ -103,4 +103,7 @@ max_usb_current=1
 [pi3]
 # for light board
 enable_uart=1
+
+# Disable temperature and low voltage overlay warnings only, safely is still ON.
```

Comment:
`safely` again :p

Reply:
Suffering from _Shooting From The Hip Syndrome_ ...